react_app/src/Cards.js

```diff
@@ -0,0 +1,284 @@
+import React, {useState} from "react";
+import { Collapse, Card, CardBody, CardText, CardTitle, Container, Row, Col } from 'reactstrap';
+// import Chevron from "./Chevron.js";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./Cards.css";
+// import TextForCards from "./TextForCards";
+
+function utcConverter(utcTime) {
+    var d = new Date(0);
+    d.setUTCSeconds(utcTime)
+    var d2 = d.getDay()
+    return d2;
+}
+
+
+let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+function dayConverter(dayNum) {
+    console.log(days[dayNum])
+    return days[dayNum]
+}
+
+function toTitleCase(str) {
+    return str.replace(/\w\S*/g, function (txt) {
+        return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+    });
+}
+
+
+const Cards = (props) => {
+    const [isOpen1, setIsOpen1] = useState(false);
+    const [isOpen2, setIsOpen2] = useState(false);
+    const [isOpen3, setIsOpen3] = useState(false);
+    const [isOpen4, setIsOpen4] = useState(false);
+    const [isOpen5, setIsOpen5] = useState(false);
+
+    if (typeof props.weather2 != "undefined" && props.weather2.daily) {
+        return (
+            <div>
```

Comment:
Consolidate this down into a single loop

Reply:
done